toml.py

```diff
@@ -25,6 +25,10 @@ def utcoffset(self, dt):
     def dst(self, dt):
         return datetime.timedelta(0)
 
+class InlineTableDict(dict):
+    def __init__(self, *args):
+        dict.__init__(self, args)
+
```

Comment:
Did you mean to use `*args` in the call to `__init__`?

If I were you, I'd suggest leaving out the method declaration entirely, or at least using `super` instead of calling the `dict` implementation directly, if you intended to avoid the ordering gotcha of `**kwargs` and `OrderedDict`.

Then an analog of the `_dict` parameter on `load` and `loads` (`_InlineTableDict`?) that allows you to override which one gets used, and you can then pass in a ordered subclass, same as you do for `_dict`. For example:

```python
class OrderedInlineTableDict(InlineTableDict, OrderedDict):
    pass
```